examples/Classification/pointcloud_classification.py

```diff
@@ -52,15 +68,16 @@
 
     optimizer.zero_grad()
```

Comment:
I'd prefer moving this to below `optimizer.step()`. That in general helps keep code modular (i.e., all backward/optimizer related code is at one place).

Reply:
done